apps/dialtone-documentation/docs/components/avatar.md

```diff
@@ -73,15 +73,22 @@ figma_url: https://www.figma.com/file/2adf7JhZOncRyjYiy2joil/DT-Core%3A-Componen
     <dt-avatar icon-name="user" icon-size="300" />
 </code-well-header>
 
-```html
+<code-example-tabs
+htmlCode='
 <div class="d-avatar d-avatar--{$size}">
     <div class="d-avatar__canvas">
         <span class="d-avatar__icon">
-            <svg>...</svg>
+        <svg>...</svg>
         </span>
     </div>
-</div>
-```
+</div>'
+vueCode='
+<dt-avatar
+    icon-name="person"
+/>
+'
+showHtmlWarning>
+</code-example-tabs>
```

Comment:
Seems like all these closing tag, not match anything

Reply:
Hmm I think it is technically closed, but yeah this is really hard to read due to the way the strings have to be. I can remove the </code-example-tabs> and close it with />. A little bit simpler.